researcher.md

```diff
@@ -4,9 +4,25 @@
 * [Back to Harvard PGP Home](http://my.pgp-hms.org)
 
 # Researcher FAQ
-### Researcher Requests for PGP Data
 
-There is a consolidated database of the Harvard PGP available for download:
+### Data Access for Harvard PGP Data
+
+The Harvard Personal Genome Project participant page, located at [my.pgp-hms.org](https://my.pgp-hms.org), holds information on participant data.
+Each Harvard Personal Genome Project participant has a profile page listing their available information.
+For example, the [profile participant for hu43860C](https://my.pgp-hms.org/profile/hu43860C) has medical information,
```

Comment:
Maybe also note that there's JSON-format data for profiles that facilitates automated retrieval of data from profile pages. For example: https://my.pgp-hms.org/profile/hu43860C.json